metrics/sinks/hawkular/driver.go

```diff
@@ -43,6 +43,9 @@ const (
 
 	nodeId string = "labelNodeId"
 
+	labelTagPrefix        = "labelTagPrefix"
```

Comment:
I would make this "labelTagPrefixOpts" or something to indicate it's not actually in the label value